includes/api/episodes/contributions.php

```diff
@@ -68,6 +68,29 @@ public function register_routes()
                 'permission_callback' => [$this, 'delete_item_permissions_check'],
             ]
         ]);
+        register_rest_route($this->namespace, '/'.$this->rest_base.'/contributions/(?P<id>[\d]+)', [
+            'args' => [
+                'id' => [
+                    'description' => __('Unique identifier for the contribution to an episode.', 'podlove-podcasting-plugin-for-wordpress'),
+                    'type' => 'integer',
+                ],
+            ],
+            [
```

Comment:
Warum hier überall `item` im Methodenname und nicht `get_contribution`, `update_contribution`, `delete_contribution` etc?

Reply:
Ich habe jetzt das item aus den Methoden entfernt. Bei get_item_permission_check (u.ä.) habe ich die Namen beibehalten, da diese auch in der Basis-Klasse so lauten.